config/initializers/simple_form.rb

```diff
@@ -14,6 +14,9 @@
   }
 
   config.wrappers :base do |b|
+    b.optional :hint
```

Comment:
Were these changes meant to be included here?

Reply:
No 🙉